release/rllib_tests/learning_tests/hard_learning_tests.yaml

```diff
@@ -1,470 +1,113 @@
-    stop:
-        time_total_s: 7200
-    config:
-        double_q: false
-        dueling: false
-        num_atoms: 1
-        noisy: false
-        n_step: 3
-        lr: .0001
-        adam_epsilon: .00015
-        hiddens: [512]
-        buffer_size: 1000000
-        exploration_config:
-            epsilon_timesteps: 200000
-            final_epsilon: 0.01
-        prioritized_replay_alpha: 0.5
-        final_prioritized_replay_beta: 1.0
-        prioritized_replay_beta_annealing_timesteps: 2000000
-        num_gpus: 1
-        num_workers: 8
-        num_envs_per_worker: 8
-        rollout_fragment_length: 20
-        train_batch_size: 512
-        target_network_update_freq: 50000
-        timesteps_per_iteration: 25000
-
-appo-pongnoframeskip-v4:
-    env: PongNoFrameskip-v4
-    run: APPO
-    # Minimum reward and total ts (in given time_total_s) to pass this test.
-    pass_criteria:
-        episode_reward_mean: 18.0
-        timesteps_total: 5000000
-    stop:
-        time_total_s: 2000
-    config:
-        vtrace: True
-        use_kl_loss: False
-        rollout_fragment_length: 50
-        train_batch_size: 750
-        num_workers: 31
-        broadcast_interval: 1
-        max_sample_requests_in_flight_per_worker: 1
-        num_multi_gpu_tower_stacks: 1
-        num_envs_per_worker: 8
-        num_sgd_iter: 2
-        vf_loss_coeff: 1.0
-        clip_param: 0.3
-        num_gpus: 1
-        grad_clip: 10
-        model:
-            dim: 42
-
-# ARS was never tested/tuned on Hopper. Maybe change to ReacherBulletEnv-v0?
-# ars-hopperbulletenv-v0:
-#    env: HopperBulletEnv-v0
-#    run: ARS
-#    # Minimum reward and total ts (in given time_total_s) to pass this test.
-#    pass_criteria:
-#        episode_reward_mean: 100.0
-#        timesteps_total: 2000000
-#    stop:
-#        time_total_s: 2000
-#    config:
-#        noise_stdev: 0.01
-#        num_rollouts: 1
-#        rollouts_used: 1
-#        num_workers: 1
-#        sgd_stepsize: 0.02
-#        noise_size: 250000000
-#        eval_prob: 0.2
-#        offset: 0
-#        observation_filter: NoFilter
-#        report_length: 3
-
-# TODO: (sven) Fix all BC-dependent learning tests for cont. actions.
-#  These seem quite hard to learn from the SAC-recorded HalfCheetahBulletEnv.
-# bc-halfcheetahbulletenv-v0:
-#    env: HalfCheetahBulletEnv-v0
-#    run: BC
-#    pass_criteria:
-#        episode_reward_mean: 400.0
-#        timesteps_total: 10000000
-#    stop:
-#        time_total_s: 3600
-#    config:
-#        # Use input produced by expert SAC algo.
-#        input: ["~/halfcheetah_expert_sac.zip"]
-#        actions_in_input_normalized: true
-
-#        num_gpus: 1
-
-#        model:
-#            fcnet_activation: relu
-#            fcnet_hiddens: [256, 256, 256]
-
-#        evaluation_num_workers: 1
-#        evaluation_interval: 3
-#        evaluation_config:
-#            input: sampler
-
-cql-halfcheetahbulletenv-v0:
+ bc-halfcheetahbulletenv-v0:
```

Comment:
Not sure I understand these changes in the learning tests file. Let me know, if this was by mistake. :)